content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -32,172 +32,42 @@ different approach.
-
-[NSX-T Container Plug-in (NCP)](https://docs.vmware.com/en/VMware-NSX-T/2.0/nsxt_20_ncp_kubernetes.pdf) provides integration between NSX-T and container orchestrators such as Kubernetes, as well as integration between NSX-T and container-based CaaS/PaaS platforms such as Pivotal Container Service (PKS) and OpenShift.
-
-### OVN (Open Virtual Networking)
-
-OVN is an opensource network virtualization solution developed by the
-Open vSwitch community.  It lets one create logical switches, logical routers,
-stateful ACLs, load-balancers etc to build different virtual networking
-topologies.  The project has a specific Kubernetes plugin and documentation
-at [ovn-kubernetes](https://github.com/openvswitch/ovn-kubernetes).
-
-### Weave Net from Weaveworks
-
-[Weave Net](https://www.weave.works/oss/net/) is a
-resilient and simple to use network for Kubernetes and its hosted applications.
-Weave Net runs as a [CNI plug-in](https://www.weave.works/docs/net/latest/cni-plugin/)
-or stand-alone.  In either version, it doesn't require any configuration or extra code
-to run, and in both cases, the network provides one IP address per pod - as is standard for Kubernetes.
+* [Amazon AWS VPC CNI plugin](https://github.com/aws/amazon-vpc-cni-k8s)
+* [Antrea](https://github.com/vmware-tanzu/antrea)
+* [Cilium](https://github.com/cilium/cilium)
+* [Cisco Application Centric Infrastructure (ACI)](https://www.cisco.com/c/en/us/solutions/data-center-virtualization/application-centric-infrastructure/index.html)
+* [Coil](https://github.com/cybozu-go/coil)
+* [Contiv-VPP](https://contivpp.io/)
+* [Flannel](https://github.com/flannel-io/flannel#flannel)
+* [Huawei CNI-Genie](https://github.com/cni-genie/CNI-Genie)
+* [Hybridnet](https://github.com/alibaba/hybridnet)
+* [Jaguar](https://gitlab.com/sdnlab/jaguar)
+* [Knitter](https://github.com/ZTE/Knitter/)
+* [Kube-OVN](https://github.com/alauda/kube-ovn)
+* [Kube-router](https://github.com/cloudnativelabs/kube-router)
+* [Microsoft Azure Virtual Network CNI plugin](https://github.com/Azure/azure-container-networking/blob/master/docs/cni.md)
+* [Multus (a Multi Network plugin)](https://github.com/k8snetworkplumbingwg/multus-cni)
+* [Nodus](https://github.com/akraino-edge-stack/icn-nodus)
+* [Noironetworks ACI CNI plugin](https://www.github.com/noironetworks/aci-containers)
+* [Nokia "Damn, Another Network Manager!" (DANM)](https://github.com/nokia/danm)
+* [OVN (Open Virtual Networking)](https://github.com/openvswitch/ovn-kubernetes)
+* [Open vSwitch](https://www.openvswitch.org/)
+* [Project Calico](https://projectcalico.docs.tigera.io/about/about-calico/)
+* [Tungsten Fabric](https://tungsten.io)
+* [VMware NSX-T](https://docs.vmware.com/en/VMware-NSX-T/index.html)
+* [Weave Net from Weaveworks](https://www.weave.works/oss/net/)
+* [cni-ipvlan-vpc-k8s](https://github.com/lyft/cni-ipvlan-vpc-k8s)
+* [fd.io](https://fd.io/)
+* [k-vswitch](https://github.com/k-vswitch/k-vswitch)
```

Comment:
To someone like me, this list has now become a list of weird terms/words. I cannot guess what "fd.io" means from the name or what "Tungsten Fabric" is. I have to click through all of them to gain an overall idea.

I am all for removing the vendor specific **details**. At the same time, I am against this change because it turns the project list into a list of names. A high-level overview (one or two sentences) for each technology can really save me a lot of time when browsing through the list for a suitable one for my new deployment.

Reply:
Duplicate comment, resolving.